dactyl-config.yml

```diff
@@ -3144,6 +3144,7 @@ pages:
             - en
 
     -   md: "@i18n/ja/references/http-websocket-apis/public-api-methods/ledger-methods/ledger.md"
+        outdated_translation: true
```

Comment:
could you explain what this addition means?

Reply:
It displays a warning on the Japanese version of the page telling users that the translated version of the page isn't up-to-date with the English version.

Reply:
very cool!